examples/annotate-variants-job.properties

```diff
@@ -0,0 +1,45 @@
+# JOB
+spring.batch.job.names=annotate-variants-job
+
+# SUBMISSION FIELDS
+input.vcf=/path/to/input.vcf
+input.vcf.id=1234
+input.vcf.aggregation=NONE
+
+input.study.name=Study name
+input.study.id=2345
+input.study.type=COLLECTION
+
+input.pedigree=
+input.fasta=/path/to/homo_sapiens/sequence.fa
```

Comment:
the vcf id is still needed for writing the annotation.tsv file. will not be used after the next PR where the annotation is written directly in mongo